components/style/parser.rs

```diff
@@ -74,3 +74,7 @@ pub fn log_css_error(input: &mut Parser, position: SourcePosition, message: &str
 pub trait Parse {
     fn parse(input: &mut Parser) -> Result<Self, ()> where Self: Sized;
 }
+
+pub trait ParseWithContext {
```

Comment:
I've changed the function signature (#14373) to include the context to avoid this problem :smile: